README.md

```diff
@@ -71,6 +71,13 @@ Some algorithms for iterating tuples, for example `tuple_fold` a fold/reduce imp
 A combination of `std::array` and `std::span` where the size is either statically known or a runtime variable
 depending on the `extent` template parameter
 
+### `generator`
+The reference implementation of `std::generator` from [P2502R2](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2022/p2502r2.pdf).
+By default, the generator and corresponding utilities are exported under the `dice::template_library::` namespace.
+If you want this generator to serve as a drop in replacement for `std::generator` until it arrives
+use `#define DICE_TEMPLATELIBRARY_GENERATOR_STD_COMPAT 1` before including the generator header. That will export
```

Comment:
Technically yes, thats why I added it as an option you have to manually enable. We will use it so that we do not have to rewrite code when `std::generator` (from `<generator>`) works on all compilers.